data_juicer/ops/base_op.py

```diff
@@ -319,15 +319,11 @@ def __init__(self, *args, **kwargs):
                 skip_op_error=self.skip_op_error,
                 op_name=self._name)
 
-    # set the process method is not allowed to be overridden
+    @classmethod
     def __init_subclass__(cls, **kwargs):
-        not_allowed_list = ['process']
```

Comment:
Maintaining `not_allowed_list = ['process']` maybe necessary, for standard OP customization?